src/App.js

```diff
@@ -1,29 +1,8 @@
-import logo from './logo.svg';
-import './App.css';
+import './index.css';
+import CalculatorDesign from './components/CalculatorDesign';
```

Comment:
- [ ] You can import this component (`CalculatorDesign`) inside the `Calculateor` component and then import the `Calculator` component in the `App.js`, kindly make sure to import the `Calculator` component here instead